bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -41,6 +41,9 @@ BPF_ARRAY(cpu_instructions, u64);
 BPF_PERF_ARRAY(cache_miss_hc_reader);
 BPF_ARRAY(cache_miss, u64);
 
+BPF_PERF_ARRAY(task_clock_hc_reader);
```

Comment:
As this is a software counter, it might be better to change the variable name to
`task_clock_sc_reader`